tools/docker/iroha-all-in-one/healthcheck.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+from iroha import Iroha, IrohaCrypto, IrohaGrpc
+
+#setup of iroha client address
+IROHA_HOST_ADDR = os.getenv('IROHA_HOST_ADDR', '127.0.0.1')
+IROHA_PORT = os.getenv('IROHA_PORT', sys.argv[2]) 
```

Comment:
I suggest to read external arguments in consistent way: use environment variables, or arguments. In all other places You are using environment variables, so let's not do exception here.

What is more if user doesn't provide arguments - the program would crash without information about usage.

Reply:
Thank you for pointing out.
The idea of doing 
`python3 healthcheck useraccount port` here is because iroha client could potentially be generated on other ports other than 50051, so people could simply pass in a different port number for the healthcheck purpose.